src/client/AuthProvider.tsx

```diff
@@ -282,15 +283,28 @@ export const AuthProvider = (props: AuthProviderProps) => {
     const redirectToSetupSAMLPage = (orgId: string, opts?: RedirectOptions) =>
         redirectTo(getSetupSAMLPageUrl(orgId, opts))
 
-    const refreshAuthInfo = async () => {
+    const refreshAuthInfo = useCallback(async () => {
         const action = await apiGetUserInfo()
         if (action.error) {
             throw new Error('Failed to refresh token')
         } else {
             dispatch(action)
             return action.user
         }
-    }
+    }, [dispatch])
```

Comment:
I'm assuming this was checked, but just verifying that this doesn't cause any weird, unexpected behavior.